test/system/220-healthcheck.bats

```diff
@@ -33,15 +33,12 @@ function _check_health {
 }
 
 @test "podman healthcheck" {
-    _build_health_check_image healthcheck_i
```

Comment:
Okay, hivemind, here's a stumper:

|    | before | after |
| ---|   ----: | ----: |
| root | 48 | 28 |
| rootless | 30 | 21 |

Time, in seconds, for `hack/bats 220` (healthcheck.bats).
* root is consistently slower than rootless
* the new code shaves 20s from root, but only 9s from rootless
  * times are now closer together, but root & rootless still differ
* the word "root" does not appear in `220-healthcheck.bats`. No conditional skips or extra tests AFAICT.
* there is a full moon

Results are consistent across O(10) runs. 

Very low priority, this is a just-curious question: any ideas?

Reply:
Rootless vs root would be different systemd daemons (the user instance vs actual PID1 systemd) which could account for some of this? I assume user instance is less busy.